### ISSUE TITLE
Developer docs

### ISSUE THREAD

**Issue by binaryfire** [romboHQ/tailwindcss-motion#24]
Awesome work guys. Will you be adding proper developer docs sometime soon? That's the only thing holding us back from trying this in our team.

**Comment by quick007** [comment]
https://docs.rombo.co/tailwind

**Comment by chukwumaokere** [comment]
Yeah its oddly enough not linked in the rombo.co/tailwind site, which made me search quite a bit to find the docs (it was linked here on Github)

**Comment by KevinGrajeda** [comment]
I added a link to the docs in the landing page 🫡
